initializr-generator/src/main/java/io/spring/initializr/generator/project/contributor/MultipleResourcesProjectContributor.java

```diff
@@ -62,7 +62,7 @@ public void contribute(Path projectRoot) throws IOException {
 		Resource root = this.resolver.getResource(this.rootResource);
 		Resource[] resources = this.resolver.getResources(this.rootResource + "/**");
 		for (Resource resource : resources) {
-			if (resource.isReadable()) {
```

Comment:
Resources aren't file when they're packaged in a JAR file and that check is breaking the build. Please run the build prior to submitting a PR.